### ISSUE TITLE
Aria-labels need to be translatable

### ISSUE THREAD

**Issue by juuso-vaadin** [vaadin-component-factory/vcf-toolbar-layout#8]
The component sets a [hard coded aria-label](https://github.com/vaadin-component-factory/vcf-toolbar-layout/blob/main/src/component/vcf-toolbar-layout.ts#L335) on the "overflow-button" which should be translatable.
